src/tasks/components/TaskRunsRow.tsx

```diff
@@ -85,6 +90,16 @@ class TaskRunsRow extends PureComponent<Props, State> {
     this.setState({isImportOverlayVisible: !this.state.isImportOverlayVisible})
   }
 
+  private handleRetry = () => {
+    const {retryTask, taskID, run} = this.props
+    try {
+      retryTask(taskID, run.id)
+      window.location.reload()
+    } catch (error) {
+      console.error(error)
+    }
```

Comment:
if we handle the error with an error notification in the thunk, this `catch` is no longer necessary

```suggestion
    }
```